wcivf/apps/elections/templates/elections/includes/_single_ballot.html

```diff
@@ -52,18 +52,28 @@ <h2>{% if postelection.is_london_assembly_additional %}Additional members{% else
               You will have one vote, and can vote for a single party list or independent candidate.
             {% else %}
               {% if postelection.winner_count and postelection.election.voting_system_id == 'FPTP' %}
-                You will have up to {{ postelection.winner_count|intword }} vote{{ postelection.winner_count|pluralize }}.
+                You will have {{ postelection.winner_count|intword }} vote{{ postelection.winner_count|pluralize }}, 
```

Comment:
I think this should be `apnumber` rather than `intword` - I think `intword` is for large numbers and `apnumber` is for smaller ones? Either way, they're not consistent in the screenshots, and I think `apnumber` is the way to fix that